automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspaceApiSpec.scala

```diff
@@ -143,13 +143,13 @@ class WorkspaceApiSpec extends TestKit(ActorSystem("MySpec")) with AnyFreeSpecLi
               case s if s.endsWith("editor") => binding.getMembers.size() shouldEqual 2
               case s if s.endsWith("genomics.serviceAgent") => binding.getMembers.size() shouldEqual 1
               case s if s.endsWith("lifesciences.serviceAgent") => binding.getMembers.size() shouldEqual 1
+              case s if s.endsWith("logging.logWriter") => binding.getMembers() shouldEqual 1
```

Comment:
Should `binding.getMembers()` be `binding.getMembers.size()`, or it doesn't matter?

Reply:
Gah, good catch, fixed.